src/persist/client-gc.ts

```diff
@@ -1,30 +1,35 @@
 import type {ClientID} from '../sync/client-id';
 import type * as dag from '../dag/mod';
-import {getClients, setClients} from './clients';
+import {ClientMap, noUpdates, updateClients} from './clients';
 
 const CLIENT_MAX_INACTIVE_IN_MS = 7 * 24 * 60 * 60 * 1000; // 7 days
 const GC_INTERVAL_MS = 5 * 60 * 1000; // 5 minutes
 
+let _latestGCUpdate: Promise<ClientMap> | undefined;
```

Comment:
My rule is that underscore is for private/protected properties/methods. Not for variable bindings

Reply:
cool yeah I'm mixing up class private vs module private (i.e. not exported).